[RLlib; docs] Fix formatting of class references. (#55764)
ray-project/ray: doc/source/rllib/rllib-examples.rst

- `Async gym vectorization, parallelizing sub-environments <https://github.com/ray-project/ray/blob/master/rllib/examples/envs/async_gym_env_vectorization.py>`__:
   Shows how the `gym_env_vectorize_mode` config setting can significantly speed up your

>    :py:class:`~ray.rllib.env.env_runner.EnvRunner` actors, if your RL environment is slow and you're

Reviewer: nice catch!

>    :py:class:`~ray.rllib.env.env_runner.EnvRunner` actors, if your RL environment is slow and you are

Reviewer 2: ```suggestion
   :py:class:`~ray.rllib.env.env_runner.EnvRunner` actors, if your RL environment is slow and you're
```
